appjs/app.js

```diff
@@ -23,6 +23,37 @@ var $ = require('jquery'),
 
 // required to make Backbone work in browserify
 Backbone.$ = $;
+//
+var oldLoadUrl = function(){
```

Comment:
`this.matchRoot()` was coming back as `not a function`, so I duplicated the function from the source code and changed the instances of `this` to `Backbone.history`.